src/wp-includes/functions.php

```diff
@@ -2079,8 +2080,29 @@ function wp_mkdir_p( $target ) {
 		$dir_perms = 0777;
 	}
 
+	/**
+	 * Fires before the directory creation.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param string $target Full path to attempt to create.
+	 * @param int    $dir_perms Directory permissions.
```

Comment:
```suggestion
	 * @param string $target    Full path to attempt to create.
	 * @param int    $dir_perms Directory permissions.
```
Minor alignment issue that I missed. 🙂